app/code/community/Mbiz/TrackingTags/Model/Filter.php

```diff
@@ -29,6 +29,9 @@ class Mbiz_TrackingTags_Model_Filter extends Mage_Core_Model_Email_Template_Filt
     protected $_customerOrders;
 
     /**
+     * Get the requested product information
+     *
+     * @param $construction
      * @return string
      */
     public function productDirective($construction)
```

Comment:
You can improve this too at the same time…

s/$construction/array $construction/